src/Microsoft.PowerShell.Commands.Utility/commands/utility/Write-Object.cs

```diff
@@ -55,10 +55,8 @@ protected override void ProcessRecord()
             {
                 enumerate = false;
             }
-            foreach (PSObject inputObject in _inputObjects) // compensate for ValueFromRemainingArguments
-            {
-                WriteObject(inputObject, enumerate);
-            }
+
+            WriteObject(_inputObjects, enumerate);
```

Comment:
this shouldn't been changed even if the fix to parameter binder is accepted. it will break the following scenario:
```
Write-Output aa,bb cc,dd
```